deploy/helm/kubecf/assets/operations/instance_groups/database.yaml

```diff
@@ -1,9 +1,31 @@
-{{- if .Values.features.external_database.enabled -}}
-
-# Remove the database instance group.
+# Remove database instance group and variables.
 - type: remove
   path: /instance_groups/name=database
 
+# Remove database links.
```

Comment:
Would it make sense to use an [entaglement](https://github.com/cloudfoundry-incubator/cf-operator/blob/master/docs/entanglements.md#native---bosh) here, with an additional service that has the same selector as the one provided by the helm chart? (As far as I know, we can't modify the existing service to have the correct annotations.)

Reply:
This is used as an "external database" from the point of view of the bosh releases.
So links are not used.